docs/cli/buttons_publish.md

```
@@ -40,20 +36,19 @@ buttons publish <name | @desk/name> [flags]
 
 ```
-  -h, --help            help for publish
-      --kind string     registry entry kind: button | drawer (default "button")
-      --source string   local source directory to publish to (else $BUTTONS_REGISTRY_URL)
+  -h, --help          help for publish
+      --kind string   registry entry kind: button | drawer (default "button")
 ```
 
 ### Options inherited from parent commands
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)